docs/src/getting-started.rst

```diff
@@ -158,8 +207,7 @@ library includes the basic classes needed to write |cdk| stacks and apps.
 
     .. group-tab:: TypeScript
 
-        Install the **@aws-cdk/cdk** package and the **@types/node** (the latter
-        is needed because we reference **process.argv** in our code):
+        Install the **@aws-cdk/cdk** and the **@types/node** packages:
```

Comment:
@types/node is actually not needed anymore

Reply:
removed